### PR TITLE
v20.2.14 release notes

### LINKED ISSUE #10708: v20.2.14 Release Notes
Prep date: Aug 13, 2021
Release date: Aug 16, 2021
SHA candidate: 7fbeb25bdbcbf8bba4553871897076f98954801a
Release Manager: @absterr08 
Issue: https://github.com/cockroachdb/cockroach/issues/68659
[Docs To Do](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-16.Updatedocs):

- [ ] Run script to generate release notes
- [ ] Update docs site
- [ ] Edit release notes - Additions/changes/omissions to the release notes should be left in this issue's comments
- [ ] Once PR is merged / release notes are live, notify @meagan-g in the #releases Slack channel
- [ ] Email releases@ with link to published release notes
- [ ] Generate release notes issues (cc @jseldess)

### PR DESCRIPTION
**Pull request by ianjevans**
Fixes #10708.

### CODE REVIEW

File: releases/v20.2.14.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: What&#39;s New in v20.2.14
+toc: true
+summary: Additions and changes in CockroachDB version v20.2.14 since version v20.2.13
+---
+
+## August 15, 2021
```
Comment (kathancox) [inline releases/v20.2.14.md:7]: Should this be August 16?

File: releases/v20.2.14.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+- [Incremental backups](../v20.2/take-full-and-incremental-backups.html) to a cloud storage location that already contains large existing backups now find their derived destination without listing as many remote files. [#67289][#67289]
+
+### SQL language changes
+
+- Added a new [`EXPLAIN` flag](../v20.2/explain.html), `MEMO`, to be used with `EXPLAIN (OPT)`. When the `MEMO` flag is passed, a representation of the optimizer memo will be printed along with the best plan. The `MEMO` flag can be used in combination with other flags such as `CATALOG` and `VERBOSE`. For example, `EXPLAIN (OPT, MEMO, VERBOSE)` will print the memo along with verbose output for the best plan. [#67778][#67778]
+
+### DB Console changes
+
+- The DB Console will now redirect the user to the originaly requested page after user login. [#67859][#67859]
```
Comment (kathancox) [inline releases/v20.2.14.md:53]: Typo here on "originally"

File: releases/v20.2.14.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+### Bug fixes
+
+- Fixed a statement buffer memory leak when using suspended portals. [#67370][#67370]
+- Correlated subqueries that couldn't be decorrelated and that have their own subqueries are now executed correctly when supported. Note that this is a very rare edge case. [#67571][#67571]
+- Fixed a very rare unexpected error from the [vectorized engine](../v20.2/vectorized-execution.html) ("index out of bounds") when evaluating the `CASE` operator. [#67785][#67785]
+- Catching up [Raft](../v20.2/architecture/replication-layer.html#raft) followers on the Raft log is now more efficient in the presence of many large Raft log entries. This helps avoid situations where Raft leaders struggle to retain leadership while catching up their followers. [#67126][#67126]
+- Fixed a rare bug in v20.2 that would happen when a lease change occured during a range merge. This bug allowed [`AS OF SYSTEM TIME`](..//v20.2/as-of-system-time.html) queries to serve inconsistent results. [#65823][#65823]
```
Comment (kathancox) [inline releases/v20.2.14.md:61]: Not sure on this one, but do we need "in v20.2"? I suppose my assumption is that all of these fixed bugs were in v20.2.

File: releases/v20.2.14.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+- Fixed an issue with statistics estimation in the [optimizer](../v20.2/cost-based-optimizer.html) that could have caused it to over-estimate the number of rows for some expressions and thus choose a sub-optimal plan. This issue could happen when multi-column statistics were used in combination with histograms, the query contained a predicate on two or more columns where the columns were hightly correlated, and the selected values were very common according to the histograms. [#68012][#68012]
+- Fixed an issue where CockroachDB could encounter an internal error or crash when performing a cast of a `NULL` [`JSON` value](../v20.2/jsonb.html) to [Geography or Geometry types](../v20.2/spatial-features.html). [#67903][#67903]
+- [`INSERT`](../v20.2/insert.html) and [`UPDATE`](../v20.2/update.html) statements which operate on larger rows are now split into batches using the `sql.mutations.mutation_batch_byte_size` setting. [#67963][#67963]
+- Fixed a rare bug that could result in a crash while creating a [`debug.zip`](../v20.2/cockroach-debug-zip.html). The bug was only possible to hit if a `debug.zip` was captured during a period of rapid lease movement. [#67727][#67727]
```
Comment (kathancox) [inline releases/v20.2.14.md:68]: The wording in the second sentence caught my ear while reading, maybe something like "It was only possible to hit the bug if...." or "... encounter the bug if..."

**Review by kathancox — APPROVED** [review]
Just a couple of nits, otherwise LGTM!

**Comment by ianjevans** [comment]
TFTR @kathancox!

**Pull request merged**